applications/workers/worker-common/src/main/kotlin/net/corda/applications/workers/workercommon/DefaultWorkerParams.kt

```diff
@@ -7,7 +7,7 @@ import net.corda.schema.configuration.ConfigDefaults
 import picocli.CommandLine.Option
 
 /** The startup parameters handled by all workers. */
-class DefaultWorkerParams {
+class DefaultWorkerParams(healthPortOverride: Int? = null) {
```

Comment:
```suggestion
class DefaultWorkerParams(healthPortOverride: Int = WORKER_MONITOR_PORT) {
```

Would maybe be simpler?

Reply:
Yeah, I like it, thanks. Will apply.